test/DacpacTool.Tests/PackageAnalyzerTests.cs

```diff
@@ -128,5 +155,16 @@ private static (FileInfo fileInfo, TSqlModel model) BuildSimpleModel()
 
             return (new FileInfo(packagePath), model);
         }
+
+        private FileInfo[] CollectAssemblyPaths()
+        {
+            var result = new List<FileInfo>();
+            var path = Path.GetDirectoryName(Path.Combine(System.Reflection.Assembly.GetAssembly(typeof(PackageAnalyzerTests)).Location));
+            result.Add(new FileInfo(Path.Combine(path, "SqlServer.Dac.dll")));
```

Comment:
Does the `SqlServer.Dac.dll` assembly contain any rules itself? Or is that more of a runtime dependency of the `SqlServer.Rules.dll` and `TSQLSmellSCA.dll`? If its the latter I think we shouldn't need to add the path to that assembly here, as it should be able to load that automagically